Makefile

```diff
@@ -12,7 +12,7 @@ GOBUILD = go build
 GOBUILD_TAGS =
 GOBUILD_ENVS = CGO_ENABLED=0
 GOBUILD_LDFLAGS =
-GOBUILD_FLAGS = -tags "$(GOBUILD_TAGS)" -ldflags "$(GOBUILD_LDFLAGS)"
+GOBUILD_FLAGS = -tags "$(GOBUILD_TAGS)" -ldflags "$(GOBUILD_LDFLAGS)" # -gcflags="all=-N -l"
```

Comment:
Please let me know the reason to comment this ?

Reply:
`-gcflags="all=-N -l"` used when I use debugger (delve). When turning this build flag, debugger can read all variables